pyiron_gui/utils/busy_check.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+import functools
+import ipywidgets as widgets
+
+__author__ = "Niklas Siemer"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
```

Comment:
```suggestion
    "Copyright 2021, Max-Planck-Institut für Eisenforschung GmbH - "
```

possibly elsewhere too, I only noticed it here.